web/themes/custom/votegov/templates/node/node--state-territory.html.twig

```diff
@@ -23,6 +23,7 @@
 {#{% set election_homepage_link = content.field_override_election_hp_link | field_value | default(content.field_election_homepage_link | field_value) | render | trim %}#}
 {% set registration_link = content.field_override_registration_link | field_value | default(content.field_registration_link | field_value) | render | trim %}
 {% set state_mail_pdf_link = content.field_override_mail_reg_link | field_value | default(content.field_mail_registration_link | field_value) | render | trim %}
+{% set military_overseas_registration_link = content.field_override_military_overseas | field_value | default(content.field_military_overseas_registra | field_value) | render | trim %}
```

Comment:
@clmedders What is the default link url supposed to be? I am not sure where this value is set. Right now there is no url when the button text is set.

Reply:
So when ever you are in the edit form for this card there is a field that has a link text... so i add a new link field to this to cover that.  the design did not have a button/link but didnt know if it would in the future so was trying to plan ahead.  I can sync up with you if we need to talk though it more!